src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -85,6 +85,7 @@
 
     private Authenticator adminAuthenticator = new NoAuthenticator();
     private boolean requireHttpsForAdminApi = false;
+    private boolean reportNearMissDisabled = false;
```

Comment:
I'm wondering if disabled by default isn't better

Reply:
(I would also name this reportNearMiss as it's simpler to understand when you're setting its value)